chapters/03_queueing_model/06_closed_form_steady_state_probs/main.tex

```diff
@@ -0,0 +1,1022 @@
+\subsubsection{Closed-form approach}
+
+This section aims to investigate the connection between Markov chains and graph
+theory by exploring alternative ways of calculating the steady state
+probabilities and considering the problem from different perspectives.
+
+\paragraph{Parameters}
+The parameters considered as inputs are:
+\begin{itemize}
+    \item the number of servers \(C\),
+    \item the threshold \(T\),
+    \item the capacity of node 1 \(N\),
+    \item the capacity of node 2 \(M\).
+\end{itemize}
+
+Additional parameters of the model are the type 1 individuals arrival rate,
+the type 2 individuals arrival rate and the service rate (\(\lambda_1,
+\lambda_2, \mu\)).
+More specifically, the way these parameters are translated into the model are:
+
+\begin{itemize}
+    \item \textbf{Number of servers (\(C\)):} Affects the weight of all edges
```

Comment:
Is there a jump here to directly talking about edges? Or have we spoken about the graph before?

Reply:
Yeah, this is the first time I'm talking about the Markov Chain as a graph